src/main/scala/io/github/sbt/tzdb/IOTasks.scala

```diff
@@ -76,8 +76,7 @@ object IOTasks {
     } yield f
 
   def providerFile(base: JFile, name: String, packageDir: String): IO[File] = IO {
-    val pathSeparator   = JFile.separator
-    val packagePath     = packageDir.replaceAll("\\.", pathSeparator)
+    val packagePath     = packageDir.replaceAll("\\.", "/")
```

Comment:
Strange, I'd expect this to be the other way around. avoid using a constant and use a system property instead.

Reply:
Thing is, `File.separator` returns a backslash `"\"` in Windows. The second argument to `replaceAll` gives special meaning to backslashes (and dollar signs), as described in:

* https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#replaceAll(java.lang.String,%20java.lang.String)
* https://docs.oracle.com/javase/7/docs/api/java/util/regex/Matcher.html#replaceAll(java.lang.String) (the effect is more clearly explained here).

Therefore, calling `replaceAll` with just a backslash as the 2nd parameter causes an exception in Windows: `java.lang.IllegalArgumentException: character to be escaped is missing`.

In Windows, everything works fine if we use a forward slash `"/"` instead of a backslash as a path separator. However, if you'd like to use the system property for the separator, we should then either escape it manually, or use https://docs.oracle.com/javase/7/docs/api/java/util/regex/Matcher.html#quoteReplacement(java.lang.String).